build.gradle

```diff
@@ -261,6 +261,12 @@ subprojects { subproject ->
 		archives javadocJar
 	}
 
+	jar {
+		manifest {
+			attributes('Implementation-Version': version)
+		}
```

Comment:
```
Manifest-Version: 1.0
Implementation-Version: 4.3.0.BUILD-SNAPSHOT
```
:+1: 

I think I'm fine with this.
Just curious if other our guys include this to their jars, too.